app.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "ClearDB Heroku Addon",
+  "description": "A web service that enables peer-to-peer video streaming between senders and receivers.",
+  "repository": "https://github.com/bean-pod/switchboard",
```

Comment:
Change the name to `switchboard`

Reply:
I’ll make the change within the next 2 hours 👍

Reply:
Done! Was able to do it on my phone 📱